spec/command_.test.sh

```diff
@@ -34,3 +34,10 @@ chmod +x $TMP/no-shebang
 $SH -c '$TMP/no-shebang'
 ## stdout: hi
 ## status: 0
+
+#### File with relative path and no shebang is executed
+echo 'echo hi' > no-shebang
```

Comment:
I'm pretty sure this will litter the repo.  `cd $TMP` first (TMP is always set in the test env)